otredis.go

```diff
@@ -59,12 +61,12 @@ func formatCommandsAsDbMethods(cmds []redis.Cmder) string {
 	return strings.Join(cmdsAsDbMethods, " -> ")
 }
 
-func doSpan(parentSpan opentracing.Span, opts *redis.Options, operationName, dbMethod string) {
+func getSpan(parentSpan opentracing.Span, opts *redis.Options, operationName, dbMethod string) (span opentracing.Span) {
```

Comment:
could you please use unnamed return? repeating names is a bad practice and named returns make it harder to understand the flow of the function. Maybe it does not really matter for such a small function but anyway better to keep the code neat.

Reply:
okay i will open a new PR with unnamed return